funnel/templates/account_merge.html.jinja2

```diff
@@ -29,11 +27,7 @@
   </ul>
 {% endmacro %}
 
-{% block contentwrapper %}
-  <div class="alert alert--error cookies-required-alert alert--dismissable">
-    <a class="alert__close" href="javascript:void(0);" aria-label="close">{{ faicon(icon='times', icon_size='title') }}</a>
-    <p class="alert__text">{% trans %}Cookies are required to perform this operation. Please enable cookies in your browser’s settings and reload this page.{% endtrans %}</p>
-  </div>
+{% block loginboxcontent %}
```

Comment:
Looks like this block's name is too specific for one use case. May need a better name, like "focuscontent".